test/generator/component-yield-multiple-in-if/_config.js

```diff
@@ -0,0 +1,14 @@
+export default {
+	solo: true,
```

Comment:
☹️

Is there a way to make the CI job fail if there are any `solo: true` tests?